example/lib/pages/enemy/enemy_page.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:bonfire/bonfire.dart';
+import 'package:example/pages/enemy/human_with_collision.dart';
+import 'package:example/pages/enemy/melee_enemy.dart';
+import 'package:example/pages/enemy/range_enemy.dart';
+import 'package:flutter/widgets.dart';
+
+class EnemyPage extends StatelessWidget {
```

Comment:
Collisions with the world objects seems off in these examples too, maybe it's in the map that needs adjusts.